icu_base.py

```diff
@@ -38,16 +37,12 @@ def _is_mingw(self):
 
     def build_requirements(self):
         if self._the_os == "Windows":
-            #self.build_requires("cygwin_installer/2.9.0@bincrafters/stable")
-            self.build_requires("msys2_installer/20161025@bincrafters/stable")
+            self.build_requires("msys2_installer/latest@bincrafters/stable")
```

Comment:
```suggestion
            self.build_requires("msys2/20190524")
```

Reply:
We don't want "latest" here?

Reply:
msys2_installer is deprecated: https://github.com/bincrafters/conan-msys2_installer

Reply:
Yes, but should we use `msys2/latest` ?

Reply:
because there is no msys2/latest : https://github.com/conan-io/conan-center-index/blob/master/recipes/msys2/config.yml